plugins/synaptics-mst/fu-self-test.c

```diff
@@ -34,7 +34,9 @@ _test_add_fake_devices_from_dir(FuPlugin *plugin, const gchar *path)
 	g_assert_no_error(error);
 	g_assert_nonnull(dir);
 
-	ret = fu_context_load_quirks(ctx, FU_QUIRKS_LOAD_FLAG_NO_CACHE, &error);
+	ret = fu_context_load_quirks(ctx,
+				     FU_QUIRKS_LOAD_FLAG_NO_CACHE | FU_ENGINE_LOAD_FLAG_HWINFO,
```

Comment:
I think we'll need the header file for `FU_ENGINE_LOAD_FLAG_HWINFO`

Reply:
Doh, I'm an idiot trying to multitask :) Will fix.